### ISSUE TITLE
Travis CI Node 4.2 builds failing

### ISSUE THREAD

**Issue by g0ddish** [mjmlio/mjml#173]
CI builds for node 4.2 are breaking with this dependency error :scream:
```
npm ERR! peerinvalid The package eslint@2.5.3 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-eslint@5.0.2 wants eslint@<2.3.0
```

**Comment by arnaudbreton** [comment]
Hey @g0ddish thanks for noticing! Looking into that and will fix shortly

**Comment by lohek** [comment]
Fix with #175
